### PR TITLE
Setup QEMU

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: scripts/ubuntu-20-provision.sh [hunk 1]
```diff
@@ -61,6 +61,15 @@ apt-get install -y --no-install-recommends \
...
   parallel
 
+# setup qemu
+apt-get install -y --no-install-recommends \
+  qemu \
+  binfmt-support \
+  qemu-user-static
+
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+docker run --rm -t arm64v8/ubuntu uname -m # test that qemu is working
```
Comment (timja) [inline scripts/ubuntu-20-provision.sh:71]:
```suggestion
```

File: scripts/ubuntu-20-provision.sh [hunk 2]
```diff
@@ -61,6 +61,14 @@ apt-get install -y --no-install-recommends \
   jq \
   parallel
 
+# setup qemu
+apt-get install -y --no-install-recommends \
+  qemu \
+  binfmt-support \
+  qemu-user-static
+
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```
Comment (dduportal) [inline scripts/ubuntu-20-provision.sh:70]: @timja shouldn't we run this command in the pipelines, to have Qemu loaded only when we need it?

Comment (timja) [reply scripts/ubuntu-20-provision.sh:70]: It just configures qemu, what would the reason of moving it be?

Comment (dduportal) [reply scripts/ubuntu-20-provision.sh:70]: I'm not comletely sure that the setting is persisted across reboot: let's check as it does not create any issue (I was overcomplicating)

**Review by dduportal — APPROVED** [review]
Let's go go go !

**Pull request merged**